### ISSUE TITLE
Chewbacca patients should be able to transform into females when they die

### ISSUE THREAD

**Issue by J-Shep** [CorsixTH/CorsixTH#177]
They can do this in TH.

I will implement this asap and then amend my commit in my Patient:isMalePatient() pull request.

**Comment by lewri** [comment]
@J-Shep I noticed your branch for this is still open. I'm not sure about the amount of work left on this but wondering if the code can still be used.

Unless this was implemented by someone else?

**Comment by SilPho** [comment]
Looks like it is already in the game as of at least 0.66. I also checked the code and it seems to match this PR.

![female_hairy](https://user-images.githubusercontent.com/22282491/190409680-cf1eeb9f-9430-404e-8a43-3307350a6ce0.png)

**Comment by lewri** [comment]
The issue wasn't their cured anim. It is their death anim. When a Chewbacca patient dies, they will always transform to males.

J-Shep's code is probably usable in principle, but they did not a fall anim glitch, that may need looking at.

**Comment by Alberth289346** [comment]
I don't see why you need to introduce a 2nd Chewbacca patient. Making the change to a male/female just before death or cure would work too, wouldn't it?

**Comment by lewri** [comment]
No, I think females have a specific death anim in contrast to males... Likely their outer shape?

**Comment by ARGAMX** [comment]
Seems this ticket has been stuck for 12 years.

If weren’t this ticket, I wouldn’t even know that hairyitis patients don’t turn into women at death. 

It seems the problem isn’t that big, and probably most players can live with it.
I’m not even sure this needs to be done or that it’s worth the effort. I don't see much value in it.

Does anyone still need this? 
Does anyone want to work on it?

**Comment by TheCycoONE** [comment]
I don't mind leaving it open as a gap. Someone identified it.
